packages/experimental/kai/src/frames/Note/NoteFrame.tsx

```diff
@@ -42,17 +66,24 @@ export const TileContent = withReactor(({ item, selected, onDelete }: TileConten
     <div className='flex flex-1 flex-col overflow-hidden'>
       <div className='flex w-full items-center mb-3'>
         {/* Title */}
-        <div className='flex flex-1 overflow-hidden'>
-          <input
-            ref={inputRef}
-            className='text-lg w-full outline-0 p-1 bg-transparent'
+        <div className='flex flex-col flex-1 overflow-hidden'>
+          <Input
+            label='Title'
+            variant='subdued'
+            value={note.title}
+            onChange={(event) => {
+              note.title = event.target.value;
+            }}
             placeholder='Title'
-            autoFocus={selected} // TODO(burdon): Not working.
-            value={item.label}
-            onChange={({ target: { value } }) => {
-              // TODO(burdon): Breaks abstraction: bubble change event instead.
-              const note = item.data! as Note;
-              note.title = value;
+            slots={{
+              label: { className: 'sr-only' },
+              root: {
+                className: 'm-0 w-full'
+              },
+              input: {
+                className: 'p-1 w-full border-0 text-xl',
+                autoFocus: true // TODO(burdon): Apply selectively?
```

Comment:
This already is applied somewhat selectively; `autoFocus` isn’t applied in iOS to prevent the mixed focus state there.

Reply:
previously i used a ref to programmatically set the focus. i assume i can do that with Input via forwardrefs?